app/src/main/java/org/oppia/app/topic/practice/practiceitemviewmodel/TopicPracticeHeaderViewModel.kt

```diff
@@ -0,0 +1,7 @@
+package org.oppia.app.topic.practice.practiceitemviewmodel
+
+import org.oppia.app.topic.practice.TopicPracticeFragment
+
+/** Header view model for the recycler view in [TopicPracticeFragment]. */
+class TopicPracticeHeaderViewModel(
```

Comment:
Write it in single line: `class TopicPracticeHeaderViewModel : TopicPracticeItemViewModel()`

Reply:
Done.